### PR TITLE
#2726 created getOrganizationImages endpoint

### LINKED ISSUE #2726: [Recruitment] - getOrganizationImages endpoint
### Desired Changes

We need a way to display the current organizations images on the frontend. To do this, we need an endpoint for it. Create a get endpoint under the route /organization/images which doesn't take in a payload. It should return both images in the organization model. 

### Acceptance Criteria
- new endpoint called getOrganizationImages at the route /organization/images
- Unit tests are added
- Manual tests are screenshotted

### Screenshots (as needed)

_No response_

### PR DESCRIPTION
**Pull request by AkulAggarwal2**
## Changes

Created the getOrganizationImages endpoint to display the current organizations images on the frontend. 

Screenshot 2024-08-21 at 9 25 48 AM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #2726

### CODE REVIEW

**Review by superhvarn — DISMISSED** [review]
LGTM

File: src/backend/src/services/organizations.services.ts [hunk 1]
```diff
@@ -109,4 +109,25 @@ export default class OrganizationsService {
...
+      where: { organizationId }
+    });
+
+    if (!organization) {
+      throw new NotFoundException('Organization', organizationId);
+    }
+
+    return {
+      applyInterestImage: organization?.applyInterestImageId,
+      exploreAsGuestImage: organization?.exploreAsGuestImageId
```
Comment (Aaryan1203) [inline src/backend/src/services/organizations.services.ts]: You shouldn't need the question mark here anymore

File: src/backend/src/services/organizations.service.ts
Comment (Aaryan1203) [inline src/backend/src/services/organizations.service.ts:1]: I think you may have accidentally added this file, you can delete it

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
Nice work, couple of small things right now

**Review by walker-sean — APPROVED** [review]
LGTM

**Pull request merged**